cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -436,6 +441,12 @@ def certificates_detail_handler(request, course_key_string, certificate_id):
 
     store = modulestore()
     if request.method in ('POST', 'PUT'):
+        if certificate_id:
+            active_certificates = CertificateManager.get_certificates(course, only_active=True)
+            if int(certificate_id) in [int(certificate["id"]) for certificate in active_certificates]:
+                # Only global staff (PMs) are able to edit active certificate configuration
+                if not GlobalStaff().has_user(request.user):
```

Comment:
@mattdrayer according to SOL-1077 only active certificate should be editable/deletable by Global staff. If certificate is not active Course Admin can also edit/delete it.